test/draw/RasterMonochromeTest.ooc

```diff
@@ -68,6 +68,11 @@ RasterMonochromeTest: class extends Fixture {
 			image2 = image2 resizeTo(size)
 			expect(image2 size == size)
 		})
+		this add("copy", func {
+			image := RasterMonochrome open(this sourceFlower)
+			image2 := image copy()
+			expect(image size == image2 size)
```

Comment:
Add
```ooc
expect(image stride == image2 stride)'
expect(image stride == image2 stride)
```
I'm pretty sure this check will fail. The `copy` function needs to call `result := This new~allocateStride(this